### DISCUSSION TITLE
how can I unroot

### DISCUSSION THREAD

**Discussion by mojtaba-mehdipour** [MustardChef/WSABuilds#395] [Q&A]
This version of Android 13 is rooted.
Now my question is how can I unroot.

Thanks

**Comment by Majinkensword** [comment]
Were you ever able to do this? Tried uninstalling magisk as I have apps that do not work with root.

**Comment by MustardChef** [comment]
> Were you ever able to do this? Tried uninstalling magisk as I have apps that do not work with root.

Get the non LTS builds and select a build without root.

Here are links to the latest non-LTS build:
- [Windows 11](https://github.com/MustardChef/WSABuilds/releases/tag/Windows_11_2407.40000.4.0)
- [Windows 10](https://github.com/MustardChef/WSABuilds/releases/tag/Windows_10_2407.40000.4.0)

**Comment by Felix14-v2** [comment]
Just install a Magisk module to hide the root from these apps.
